templates/repo/issue/new_form.tmpl

```diff
@@ -129,28 +135,6 @@
 						</a>
 					{{end}}
 				</div>
-
-			<!-- input id="assignee_ids" name="assignee_ids" type="hidden" value="{{.assignee_id}}">
```

Comment:
Hello, I don't think these comments are usefull, so I suggest to remove them, If it have some special reason that should be kept. please tell me, I will recover them. thanks